sdk/objectanchors/Azure.MixedReality.ObjectAnchors/src/Vector3Extensions.cs

```diff
@@ -0,0 +1,23 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace Azure.MixedReality.ObjectAnchors
+{
+    using System;
+    using System.Numerics;
+
+    internal static class Vector3Extensions
+    {
+        private const float normalizationAccuracy = 1.0e-4f;
+
+        /// <summary>
+        /// Determines whether the specified vector is normalized.
+        /// </summary>
+        /// <param name="vector">The vector.</param>
+        /// <returns><c>true</c> if the specified vector is normalized; otherwise, <c>false</c>.</returns>
+        public static bool IsNormalized(this Vector3 vector)
```

Comment:
I'm surprised System.Numerics doesn't have this already.

Reply:
Me too, but they expose the code to do the normalization...? Wacky.